packages/contracts/src/EntryPointForwarder.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+contract EntryPointForwarder {
```

Comment:
Should we configure the contract to specify the EP address? This one is too generic and can be misused to forward requests elsewhere, no?

Reply:
> This one is too generic and can be misused to forward requests elsewhere, no?

yes, but It's just easier this way and it might come in handy if there will be issues with other contracts in the future